docs/source/installation.rst

```diff
@@ -0,0 +1,106 @@
+.. _installation:
+
+Installation instructions
+=========================
+
+Note: T3 was only tested on Linux (Ubuntu_ 18.04.1 LTS) and Mac machines. We don't expect it to work on Windows.
+T3 can be installed on a server, as well as on your local desktop / laptop, submitting jobs to the server/s.
+Note that T3 requires installation of `ARC <https://reactionmechanismgenerator.github.io/ARC/installation.html>`_, Automated Rate calculator.
+
+
+.. _path:
+
+Clone and setup path
+^^^^^^^^^^^^^^^^^^^^
+
+- Download and install the `Anaconda Python Platform`__ for Python 3.7 or higher if you haven't already.
+- Get git if you don't have it already by typing sudo apt-get install git in a terminal.
+- Clone T3's repository to by typing the following command in the desired folder (e.g., under `~/home/Code/`)::
+
+    git clone https://github.com/ReactionMechanismGenerator/T3.git
+			  
+- Add T3 to your local path in .bashrc (make sure to change `~/Path/to/T3/` accordingly)::
+
+    export PYTHONPATH=$PYTHONPATH:~/Path/to/T3/
+
+__ anaconda_
+
+
+Install dependencies
+^^^^^^^^^^^^^^^^^^^^
+
+- Install the latest version of ARC on the same machine that T3 is installed on 
+  by following `ARC's installation instructions 
+  <https://reactionmechanismgenerator.github.io/ARC/installation.html>`_. 
+  Make sure to add ARC to your PATH and PYTHONPATH variables as well as define servers as 
+  explained in ARC's documentation. 
+- If you'd like to use `AutoTST <https://github.com/ReactionMechanismGenerator/AutoTST>`_ in ARC (optional),
+  clone it in a separate folder and add it to your PYTHONPATH just as well.
+- Create the Anaconda environment for T3::
+
+    conda env create -f environment.yml
+
+  Activate the T3 environment every time before you run T3::
+
+     source activate t3_env
+
+
+Add T3 aliases to your .bashrc (for convenience)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Some optional, but convenient, aliases are listed below (make sure to change `~/Path/to/T3/` accordingly).
+Add these to your ``.bashrc`` file, which can be edited by typing, e.g., ``nanoe ~./bashrc``)::
```

Comment:
I think it should be `nano`, not `nanoe`

Reply:
Indeed, another sloppy typo on my end..